### PR TITLE
Feature/condo/sberdoma 441 Ticket Classifier by create schema ( remain from 3 selects ticket classifier)

### PR DESCRIPTION
**Pull request by dkoviazin**
TicketClassifier to create schema
PS: It's a part of the 3-LVL classifier select that remains

### CODE REVIEW

**Comment by AntonAL** [comment]
Please, add tests for `User.isSupport`

File: apps/condo/domains/ticket/schema/TicketClassifier.test.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    describe('User', () => {
+        it('cant create', async () => {
+            const admin = await makeLoggedInAdminClient()
+            const [_, userAttrs] = await createTestUser(admin)
+            const client = await makeLoggedInClient(userAttrs)
+            await expectToThrowAccessDeniedErrorToObj(async () => {
+                await createTestTicketClassifier(client)
+            })
+        })
+        it('can read', async () => {
```
Comment (AntonAL) [inline apps/condo/domains/ticket/schema/TicketClassifier.test.js:21]: Please, inspect `parent` field

File: apps/condo/domains/ticket/utils/clientSchema/search.ts [hunk 2]
```diff
@@ -20,7 +20,7 @@ const GET_ALL_SOURCES_QUERY = gql`
 
 // TODO(pahaz): add organization relation to existing classifiers
```
Comment (AntonAL) [inline apps/condo/domains/ticket/utils/clientSchema/search.ts:21]: It seems, like `organisation` relation was removed from `TicketClassified` and this comment is now orphaned.

File: apps/condo/domains/ticket/schema/TicketClassifier.test.js (hunk 1 above)
Comment (dkoviazin) [reply apps/condo/domains/ticket/schema/TicketClassifier.test.js:21]: Done

File: apps/condo/domains/ticket/utils/clientSchema/search.ts (hunk 2 above)
Comment (dkoviazin) [reply apps/condo/domains/ticket/utils/clientSchema/search.ts:21]: The future of classifiers is undefined, so I shall not remove this todo

**Review by pahaz — APPROVED** [review]

File: apps/condo/domains/ticket/access/TicketClassifier.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
+/**
+ * Generated by `createschema ticket.TicketClassifier 'organization?:Relationship:Organization:CASCADE;name:Text;parent?:Relationship:TicketClassifier:PROTECT;'`
+ */
+
+async function canReadTicketClassifiers ({ authentication: { item: user } }) {
+    if (!user) return false
+    if (user.isAdmin) return {}
+    return {
+        // TODO(codegen): write canReadTicketClassifiers logic!
```
Comment (pahaz) [inline apps/condo/domains/ticket/access/TicketClassifier.js:9]: It's better if you cant read classification of another organization

**Pull request merged**